### ISSUE TITLE
Hidden panel on storybook

### ISSUE THREAD

**Issue by Pooya-Oladazimi** [ts4nfdi/terminology-service-suite#138]
Sometimes the panel gets hidden in the storybook (if one closes it accidentally). Users can restore it by pressing "d" on the keyboard. 
Tested on Firefox.

![Screenshot from 2024-09-27 12-54-59](https://github.com/user-attachments/assets/6a74e4f7-fc65-47c8-969f-d6347db7a964)

Might be better to document it somewhere on the page as a hint for users. (like: in case you are not able to see the panel, press d)

**Comment by VincentKneip** [comment]
There's also a button on the top right corner to show the panel. But it's rather inconspicuous.

**Comment by jusa3** [comment]
Yes, I too have had the "where the hell is the panel" situation. We should add it to **How to use this Storybook** in the overview.
